bubble_sort/Python/Bubblesort.py

```diff
@@ -0,0 +1,11 @@
+def Bubble_sort(arr):
+    for j in range(n-1,0,-1):
+        for i in range (j):
+            if arr[i]>arr[i+1]:
+                arr[i],arr[i+1]=arr[i+1],arr[i] //swapping elements 
+                
+arr=[45,78,3,45,76,78,23]
+n=len(arr)
+Bubble_sort(arr)
+print(arr)
+            
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpqb3ldfvc/bubble_sort/Python/Bubblesort.py
+++ b/tmp/tmpqb3ldfvc/bubble_sort/Python/Bubblesort.py
@@ -8,4 +8,4 @@
 n=len(arr)
 Bubble_sort(arr)
 print(arr)
-            
+
```